EC-Switch-Seclog.py

```diff
@@ -150,6 +150,12 @@ def main(argv):
         print("Exiting...")
         sys.exit(1)
 
+    if pre_flight_cfn(account_session) == False:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)

Similar issue at line numbers 184 and 203.